Sources/MessagingPush/PushHandling/PushEventHandlerProxy.swift

```diff
@@ -29,9 +29,9 @@ class PushEventHandlerProxyImpl: PushEventHandlerProxy {
     private var nestedDelegates: [String: PushEventHandler] = [:]
 
     func addPushEventHandler(_ newHandler: PushEventHandler) {
-        // TODO: this line below seems fragile. If we change the class name, this could break.
+        // this line below seems fragile. If we change the class name, this could break.
```

Comment:
TODO is removed because [we have a ticket](https://linear.app/customerio/issue/MBL-70/assert-that-sdk-does-not-forward-push-click-events-to-itself) that will take care of this.

Reply:
this is one of those instance where I see good value in having a GH issue referenced in the code to make it clear that we have it on our radar.